.github/workflows/aws.yml

```
@@ -0,0 +1,63 @@
+
+    - name: Set Env
+      id: env
+      run: |
+        if [[ "${GITHUB_REF}" == "refs/heads/production" ]]; then
+          echo "TAG=prod" >> $GITHUB_OUTPUT
+        else
+          echo "TAG=staging" >> $GITHUB_OUTPUT
+        fi
+
+    - name: Login to Docker Hub
+      uses: docker/login-action@v2
```

actionlint (1.7.12)

[error] the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)